pkg/platform/api/buildplanner/request/createproject.go

```diff
@@ -1,13 +1,11 @@
 package request
 
-import "github.com/ActiveState/cli/pkg/platform/runtime/buildexpression"
-
-func CreateProject(owner, project string, private bool, expr *buildexpression.BuildExpression, description string) *createProject {
+func CreateProject(owner, project string, private bool, expr []byte, description string) *createProject {
```

Comment:
It'd be more helpful from a debugging and logging perspective to have the `expr` argument as a string. What do you think about changing it to a string?

Reply:
This has since been updated. The argument is still bytes, but what we send on the wire is a string.